tools/apigee/staging.env

```diff
@@ -1,4 +1,5 @@
 DC_API_TARGET=staging.api.datacommons.org
 APIGEE_ORGANIZATION=datcom-apigee-dev
+APIGEE_API_PRODUCT=api-staging
```

Comment:
can update this to `datacommons-api-staging`

Reply:
Done. Updated prod to `datacommons-api` as well.